src/library/browse/browsetablemodel.cpp

```diff
@@ -133,8 +135,8 @@ BrowseTableModel::~BrowseTableModel() {
 }
 
 int BrowseTableModel::columnIndexFromSortColumnId(TrackModel::SortColumnId column) {
-    if (column < TrackModel::SortColumnId::SORTCOLUMN_ID_MIN|
-        column > TrackModel::SortColumnId::SORTCOLUMN_ID_MAX) {
+    if (column<TrackModel::SortColumnId::SORTCOLUMN_ID_MIN ||
+                column>TrackModel::SortColumnId::SORTCOLUMN_ID_MAX) {
```

Comment:
Are you using out pre-commit hooks? I thought it enforces spaces around operators. In any case, please fix.

Reply:
No, I'm on Windows, where this is not available.
I used git apply with the patch file from CI.

Reply:
I think it is available now, but you need to install `clang-format` separately.